packages/react-components/src/components/VideoGallery/LocalVideoCameraCycleButton.tsx

```diff
@@ -3,18 +3,18 @@
 
 import { IButtonStyles, IconButton } from '@fluentui/react';
 import React from 'react';
-import { OptionsDevice } from '@internal/react-components/src/components/DevicesButton';
+import { OptionsDevice } from '../..';
 
 /**
  * @internal
  */
-export interface LocalVideoCameraButtonProps {
+export interface LocalVideoCameraCycleButtonProps {
   /** Array of cameras available to the user. */
-  cameras: OptionsDevice[];
+  cameras?: OptionsDevice[];
```

Comment:
Why are we making these props optional?

Reply:
@dmceachernmsft good to mention here as a @TODO that these props should be made required 
(I am assuming they will be in your follow up PR)

Reply:
To avoid making them optional here, anywhere using this now can do
```
<LocalVideoCameraCycleButton cameras={undefined as any} currentCamera={undefined as any} etc.. />
```